src/vs/workbench/contrib/debug/common/debugModel.ts

```diff
@@ -1340,23 +1359,29 @@ export class DebugModel implements IDebugModel {
 		return this.exceptionBreakpoints;
 	}
 
+	getExceptionBreakpointsForSession(sessionId?: string): IExceptionBreakpoint[] {
```

Comment:
I've changes some usages of `getExceptionBreakpoints` with the new `getExceptionBreakpointsForSession` when appropriate.

The usage in `logDebugSessionStart` in `debugTelemetry.ts`, I'm not sure about what is the preferred one to use.